packages/engine/src/networking/serialization/DataReader.ts

```diff
@@ -3,6 +3,7 @@ import { TypedArray } from 'bitecs'
 import { NetworkId } from '@xrengine/common/src/interfaces/NetworkId'
 import { UserId } from '@xrengine/common/src/interfaces/UserId'
 
+import { FLOAT_PRECISION_MULT, QUAT_MAX_RANGE } from '../../common/constants/MathConstants'
```

Comment:
These constants aren't useful in any other context, so it seems more appropriate to define them somewhere in the scope of these serialization modules to minimize clutter.